telegram/ext/filters.py

```diff
@@ -1965,16 +1965,16 @@ def remove_chat_ids(self, chat_id: SLT[int]) -> None:
 
     class sender_chat(_ChatUserBaseFilter):
         # pylint: disable=W0235
-        """Filters messages to allow only those which are from a specified sender chats chat ID or
+        """Filters messages to allow only those which are from a specified sender chat's chat ID or
```

Comment:
👍 

We should add new filters for the new message attributes, too

Reply:
yep doing that today